tests/System.Text.Primitives.Tests/Encoding/TextEncoderTestHelper.cs

```diff
@@ -526,20 +524,6 @@ public static bool IsSupportedCodePoint(uint codePoint)
                 return false;
             if (codePoint >= TextEncoderConstants.Utf16HighSurrogateFirstCodePoint && codePoint <= TextEncoderConstants.Utf16LowSurrogateLastCodePoint)
                 return false;
-            if (codePoint >= 0xFDD0 && codePoint <= 0xFDEF)
```

Comment:
Again, I vaguely recall that if you generate strings with these characters, you end up getting intermittent crashes in the tests.